package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "api-server",
+  "version": "1.0.0",
+  "description": "* Created by Steve Baldock",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs",
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Hey Steve! Your tests aren't able to run right now because of this line! You need to switch this to:
```
"test": "jest --coverage",
"test-watch": "jest --watchAll --coverage"
```